benchmarks/bench_fused_rmsnorm_silu.py

```
@@ -0,0 +1,59 @@
+import flashinfer
+from flashinfer.testing.utils import bench_gpu_time
+
+C_VALUES = [64, 128, 160, 256, 320, 512, 640, 1024]
+TOKEN_VALUES = [1560, 6240, 24960, 99840, 399360]
+
+
+def main():
+    device = torch.cuda.current_device()
+    print(f"GPU: {torch.cuda.get_device_name(device)}")
+    print(f"SM: {torch.cuda.get_device_capability(device)}")
+    print(f"Timing: CUPTI, dry_run=10, repeat=30")
```

Ruff (0.15.5)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)